src/Sentry/Protocol/IScope.cs

```diff
@@ -129,29 +85,22 @@ public User User
         /// </remarks>
         /// <example> { "fingerprint": ["myrpc", "POST", "/foo.bar"] } </example>
         /// <example> { "fingerprint": ["{{ default }}", "http://example.com/my.url"] } </example>
-        public IEnumerable<string> Fingerprint => InternalFingerprint ?? Enumerable.Empty<string>();
+        IEnumerable<string> Fingerprint { get; set; }
 
         /// <summary>
         /// A trail of events which happened prior to an issue.
         /// </summary>
         /// <seealso href="https://docs.sentry.io/platforms/dotnet/enriching-events/breadcrumbs/"/>
-        public IEnumerable<Breadcrumb> Breadcrumbs => InternalBreadcrumbs ??= new ConcurrentQueue<Breadcrumb>();
+        ICollection<Breadcrumb> Breadcrumbs { get; }
```

Comment:
Collection allows for Add, this is not expected to let people add to directly (we have those `AddBreadcrumbs` API.
```suggestion
        IEnumerable<Breadcrumb> Breadcrumbs { get; }
```